static/src/javascripts/projects/common/modules/commercial/support-utilities.js

```diff
@@ -3,14 +3,43 @@
 import config from 'lib/config';
 import { getSync as geolocationGetSync } from 'lib/geolocation';
 
-const supportBaseURL = 'https://support.theguardian.com/uk';
-const supportButtonCaption = 'Support the Guardian';
-const useSupportDomain = (): boolean =>
+const liveInUk =
     config.get('switches.ukSupporterTrafficToNewSupportFrontend') &&
     geolocationGetSync() === 'GB';
+const liveInUs =
+    config.get('switches.usSupporterTrafficToNewSupportFrontend') &&
+    geolocationGetSync() === 'US';
+const supportBaseURL = liveInUs
+    ? 'https://support.theguardian.com/us/contribute'
+    : 'https://support.theguardian.com/uk';
+const supportButtonCaption = 'Support the Guardian';
+
+const useSupportDomain = (): boolean => liveInUk || liveInUs;
 const selectBaseUrl = (defaultUrl: string = supportBaseURL): string =>
     useSupportDomain() ? supportBaseURL : defaultUrl;
 const selectEngagementBannerButtonCaption = (defaultCaption: string) =>
     useSupportDomain() ? supportButtonCaption : defaultCaption;
 
-export { useSupportDomain, selectBaseUrl, selectEngagementBannerButtonCaption };
+const liveBlogEpicSupportCtaLinkPara = (
```

Comment:
I'd be more inclined to export `liveInUk` and `liveInUs` and then have this logic in `acquisitions-copy.js`. Then all the copy stays in one place. Also, you wouldn't have the redundant `membershipUrl` and `contributionsUrl` params on the `liveblog` function

Reply:
Ok, testing this now...